ivy/functional/backends/numpy/experimental/elementwise.py

```diff
@@ -429,3 +429,13 @@ def xlogy(
 
 def real(x: np.ndarray, /, *, out: Optional[np.ndarray] = None) -> np.ndarray:
     return np.real(x)
+
+
+def ldexp(
+    x: np.ndarray,
+    i: Union[np.ndarray, int, list, tuple],
```

Comment:
It doesn't change much really but only torch uses the notation `x` and `i` for the inputs, might be better to stick with `x1` and `x2` as is the case numpy and jax ? I see you've used the latter notation for the testing when you passed `x1=x[0]` and `x2=x[1]` , you've also used that in the jax backend. In general you want to keep consistency in input format across the backends. It's at the front-end level that we mimic the patterns of each framework separately